### DISCUSSION TITLE
Theme Contributions

### DISCUSSION THREAD

**Discussion by aUnicornDev** [aUnicornDev/tabsMonster#24] [General]
Have been getting a lot of Theme contributions lately and I wanted to update the themes.md file for new contributions 

- The monsterName and the level are not the only things that will be visible on the UI
- We also need a monsterQuote for every monster present 
- Moving ahead we will need square images for characters as well but that can be ignored for now.

It would be great if the new pull request can have the quote column as well.

An example quote would be Captain America saying "I can do this all day",

**Comment by vanzasetia** [comment]
I have several questions:
* The monster quote can anything that related to the character, right? Like for example, Mr Krabs like to say, "I love money!".
* Does the quote need to be wrapped with double quotes?
* Also to make it more clear, could you update the theme example that you created with quote?

**Reply by aUnicornDev** [reply]
Hi @vanzasetia ,
 - yes the quote can be any famous line from the character
 - No, it doesn't need to be wrapped in double quotes, that will be handles on the UI
 -  I've added the changes to both THEMES.md as well as README.md for better clarity. 
 
Thank you pointing these things out.
